nbgrader/docs/source/user_guide/tests.yml

```diff
@@ -0,0 +1,310 @@
+#kernel_name:
```

Comment:
Is this commented out config needed?

Reply:
They were meant to serve as a template for new kernels
We removed them to make it cleaner